common/components/ConfirmationModal/components/Body/Body.scss

```diff
@@ -0,0 +1,104 @@
+@import 'common/sass/variables';
+
+.Body {
+  margin: auto;
+  max-width: 35rem;
+  width: 100%;
+  @media screen and (min-width: $screen-sm) {
+    min-width: 32rem;
+  }
+}
+
+.Testnet-warn {
+  text-align: center;
+  margin: 0;
+}
+
+.Address {
```

Comment:
I'd be very careful about class name collisions here. Maybe be a little more specific, like `.ConfAddress` or something like that.